deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,22 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csi defines CSI Driver settings applied per cluster.
+  csi:
+    readAffinity:
+      # Set enabled to true to enable read affinity.
+      # Refer to this [section]https://github.com/rook/rook/blob/master/Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md#enable-read-affinity-for-rbd-volumes
```

Comment:
Links aren't rendered in the yaml, how about this? We want to link to the rendered docs as well, instead of the .md.
```suggestion
# For more details see this doc:
# https://rook.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/#enable-read-affinity-for-rbd-volumes
```